### DISCUSSION TITLE
Is it possible to ignore nullable values if the have already been validated?

### DISCUSSION THREAD

**Discussion by jepzen** [fluentassertions/fluentassertions#2822] [Q&A] [answered]
Example

![image](https://github.com/user-attachments/assets/c7966497-5f86-4afa-99a9-2a7822567eb5)

I already made sure Value is not null but I still get a warning. Using JetBrains Rider

**Comment by eNeRGy164** [comment, marked as answer]
This is a valid case for the [! (null-forgiving) operator](https://learn.microsoft.com/dotnet/csharp/language-reference/operators/null-forgiving?WT.mc_id=AI-MVP-5004268)

You can change it into `result.Value!` as you already know it is not `null`
